packages/sku-codemod/src/transform/runner.ts

```diff
@@ -0,0 +1,178 @@
+import { globbySync } from 'globby';
+import prompts from 'prompts';
+import { dirname, join, resolve } from 'node:path';
+import { CODEMODS } from '../utils/constants.js';
+import { fileURLToPath } from 'node:url';
+import { Worker } from 'node:worker_threads';
+import os from 'node:os';
+import picocolors from 'picocolors';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export const transformerDirectory = join(__dirname, '../', 'codemods');
+
+type Options = {
+  dry?: boolean;
+  print?: boolean;
+};
+
+type JobWorkerData = {
+  transformerPath: string;
+  options: Options;
+  jobs: Array<{
+    filePath: string;
+  }>;
+};
+
+export const runTransform = async (
+  transform: string,
+  path: string,
+  options: Options,
+): Promise<void> => {
+  let transformer = transform;
+  console.log('path', path);
+  let directory = path;
```

Comment:
This is perhaps a style choice, but I can assure you that `let` is needed far less than you think, and can usually be written with `const` instead. I find you can catch encapsulations better when trying to write with less reassignable variables, and keeps code readable in a linear format. Debugging with `let` usually required more mental effort compared to using variables that can't be reassigned. 

Here's a suggestion for this code:

```suggestion
  const transformer = transform || (await promptTransformationType())
  const directory = path || (await promptDestination());
  console.log('path', directory);
```

Reply:
I'll look at refactoring this.